src/Sylius/Behat/Element/Admin/PromotionCoupon/GenerateFormElement.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Admin\PromotionCoupon;
+
+use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
+
+final class GenerateFormElement extends BaseFormElement implements GenerateFormElementInterface
```

Comment:
As the coupon generation form is only used in one place, I do not see the benefit of extracting these methods from page to element 🤔